doc/devel/documenting_mpl.rst

```diff
@@ -8,8 +8,17 @@ Getting started
 ===============
 
 The documentation for matplotlib is generated from ReStructured Text using the
-Sphinx_ documentation generation tool. Sphinx-1.3 or later and numpydoc 0.4 or
-later is required.
+
+Sphinx_ documentation generation tool.
+The requirements are as follows (https://github.com/matplotlib/matplotlib/blob/master/doc-requirements.txt)
```

Comment:
This file only list the pip-installable dependencies, as these can be installed directly using:
pip install -r doc-requirements.txt

Unfortunalety, we also have dependencies that need to be installed separetaly (graphviz).